tests/Oryx.Integration.Tests/Node/NodeTestUsingZippedNodeModules.cs

```diff
@@ -122,5 +123,46 @@ public async Task Node_CreateReactAppSample_zippedNodeModules(string nodeVersion
                 });
         }
 
+        [Fact]
+        public async Task BuildsAndRunsNodeApp_WhenPruneDevDependenciesIsTrue_AndNodeModulesAreCompressed()
+        {
+            // Arrange
+            // Use a separate volume for output due to rsync errors
+            var appOutputDirPath = Directory.CreateDirectory(Path.Combine(_tempRootDir, Guid.NewGuid().ToString("N")))
+                .FullName;
+            var nodeVersion = "10";
+            var appOutputDirVolume = DockerVolume.CreateMirror(appOutputDirPath);
+            var appOutputDir = appOutputDirVolume.ContainerDir;
+            var appName = "webfrontend";
+            var volume = CreateAppVolume(appName);
+            var appDir = volume.ContainerDir;
+            var runAppScript = new ShellScriptBuilder()
+                .AddCommand($"oryx -appPath {appOutputDir} -bindPort {ContainerPort}")
+                .AddCommand(DefaultStartupFilePath)
+                .ToString();
+            var buildScript = new ShellScriptBuilder()
+               .AddCommand(
+                $"oryx build {appDir} -i /tmp/int -o /tmp/out --platform nodejs " +
+                $"--platform-version {nodeVersion} -p {NodePlatform.CompressNodeModulesPropertyKey}=zip" +
+                $" -p {NodePlatform.PruneDevDependenciesPropertyKey}=true")
+               .AddCommand($"cp -rf /tmp/out/* {appOutputDir}")
```

Comment:
why not using -o {appOutputDir} directly

Reply:
This is related to rsync errors that we see especially on windows machines. This is a workaround for it.